core-js-fix.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
I don't understand why this was necessary.  Under what circumstances would this directory be present, and when would it not be?

Reply:
I found it in multiple places in webapp run scripts:

https://github.com/Exabyte-io/exabyte-web-app/blob/master/run-meteor.sh#L230

https://github.com/Exabyte-io/exabyte-saltstack/blob/1b40cf33b9c27f57989823125df8b0172d17accb/salt/web/webapp/files/install-npm-dependencies.sh#L16

As I understood, it's because of babel:
```
# babel/runtime moved core-js into babel/runtime-corejs2, hence the below
```

But one of the packages still rely on `core-js`.

Probably, @timurbazhirov could help to clarify it.

Reply:
I've added postinstall script to make it patch this issue right after dependency installation.

Reply:
Ok, sounds legit to me.

Reply:
👍